docs/stencil-docs/developing-further/catalog-price-object.md

```diff
@@ -193,79 +154,75 @@ lineNumbers: true
+* `sale_price_without_tax` - this property maps to the product’s effective price in the control panel's **Sale Price** field. The `sale_price_without_tax` property will directly expose the sale price set on a base product, variant, or price record.
+* `non_sale_price_without_tax` - this property displays the regular store price.
 
-Properties noted will only be displayed if a **Sale Price** is set on the product.
+Properties noted will only be displayed if a sale price is set on the product.
 
-### Prices and Conditional Logic Example
+### Prices and conditional logic example
 
 Stencil structures product prices for backward compatibility with the BigCommerce platform's traditional treatment of prices. This behavior enables you to add logic that determines whether to display a strikeout price on the storefront.
 
-The example below tests for the presence of the `sale_price_without_tax` property (you could also look for the `sale_price_with_tax` property). The presence of the `sale_price_without_tax` property means that the product has been given a **Sale Price**.
+The example below tests for the presence of the `sale_price_without_tax` property. (You could also look for the `sale_price_with_tax` property.) The presence of the `sale_price_without_tax` property means that the product has been given a sale price.
 
-With the logic below, the page will display the standard store price with a strikeout across it, placed next to the **Sale Price**, indicating the current selling price of the product.
+With the logic below, the page will display a strikeout standard price next to the **Sale Price** field, indicating the current selling price of the product.
 
 <div class="HubBlock-header">
     <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">Conditional Logic </div>
+        <div class="HubBlock-header-name">Conditional logic </div>
     </div><div class="HubBlock-header-subtitle"></div>
 </div>
 
 <!--
-title: "Conditional Logic "
+title: "Conditional logic "
 subtitle: ""
 lineNumbers: true
 -->
 
 ```
   {{#or price.sale_price_without_tax price.sale_price_with_tax}}
-     ... [code to display on-sale strikeout pricing or content for an on sale product]  ...
+     ... [code to display on-sale strikeout pricing or content for a discounted product]  ...
   {{/or}}
 ```
 
-## Catalog Price Object Examples Including and Excluding Tax
+## Catalog price object examples including and excluding tax
 
-Depending on how the store has been set up in the Control Panel’s **Store Setup** > **Tax** > **Configure Tax Display Settings**, the *Catalog Price Object* properties may represent prices in the following ways:
+Depending on how the store has been set up in the control panel’s **Store Setup** > **Tax** > **Tax Settings** > **Configure Tax Display Settings**, the `price` object will display prices in one of the following ways:

```

Comment:
You can hyper link the navigation path using `https://login.bigcommerce.com/deep-links/manage/` -- when someone navigates to the url, they'll login to their control panel, then be taken to the control panel page specified by the part of the url after `manage`.

Reply:
```suggestion
Depending on the store’s  [**Tax Display Settings**](https://login.bigcommerce.com/deep-links/manage/settings/tax/settings), the `price` object will display prices in one of the following ways:

```